keras/preprocessing/image.py

```diff
@@ -152,12 +146,24 @@ def __init__(self,
                  height_shift_range=0.,
                  shear_range=0.,
                  horizontal_flip=False,
-                 vertical_flip=False):
+                 vertical_flip=False,
+                 dim_ordering='th'):
         self.__dict__.update(locals())
         self.mean = None
         self.std = None
         self.principal_components = None
         self.lock = threading.Lock()
+        assert dim_ordering in {'tf', 'th'}, 'dim_ordering must be in {tf, th}'
```

Comment:
I'd rather go with an `Exception`, and specify `one of "tf" or "th"`, to make it clear these are strings. Maybe specifying the meaning of each convention would be nice too.